docs/tutorials/geomesa-quickstart-fsds.rst

```diff
@@ -0,0 +1,256 @@
+GeoMesa FSDS Quick Start
```

Comment:
i think you need to add to the index.rst so it shows up

Reply:
fixed